rust/test

```diff
@@ -1048,7 +1048,7 @@ test_rest_endpoints() {
     curl -s http://localhost:${port}/summary > output.json
 
     blockchain_length=$(cat output.json | jq -r .blockchainLength)
-    assert '100' $blockchain_length
+    assert '90' $blockchain_length
```

Comment:
@Isaac-DeFrain I'm bringing back this convo after making the last one disappear after a rebase. So, why do we get a different answer, now, with more log files, on this line?